assignments/objects.js

```diff
@@ -3,39 +3,42 @@
 // ==== Challenge 1: Writing Objects ==== 
 // HR needs some information on the new interns put into a database.  Given an id, email, first name, and gender. Create an object for each person in the company list:
 
-// 1,mmelloy0@psu.edu,Mitzi,F
-// 2,kdiben1@tinypic.com,Kennan,M
-// 3,kmummery2@wikimedia.org,Keven,M
-// 4,gmartinson3@illinois.edu,Gannie,M
-// 5,adaine5@samsung.com,Antonietta,F
 
-// Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
 
-// Write your intern objects here:
-
-
-// ==== Challenge 2: Reading Object Data ==== 
-// Once your objects are created, log out the following requests from HR into the console:
+class Intern {
+    constructor(id, email, name, gender) {
+        this.id = id
+        this.email = email
+        this.name = name
+        this.gender = gender
+    }
+}
 
-// Mitzi's name
 
-// Kennan's ID
 
-// Keven's email
+// Example format of an intern object: 1,examples@you.edu,Example,F
+const example = {
+    "id": 0,
+    "name": "Example",
+    "email": "examples@you.edu",
+    "gender": "F"
+}
 
-// Gannie's name
+// Write your intern objects here:
+// 1,mmelloy0@psu.edu,Mitzi,F
+// 2,kdiben1@tinypic.com,Kennan,M
+// 3,kmummery2@wikimedia.org,Keven,M
+// 4,gmartinson3@illinois.edu,Gannie,M
+// 5,adaine5@samsung.com,Antonietta,F
 
-// Antonietta's Gender
+const mitzi = new Intern(1, 'melloy0@psu.edu', 'Mitzi', 'F');
+const antonietta = new Intern(5, 'adaine5@samsung.com', 'Antonietta', 'F')
+console.log(mitzi.name)
 
-// ==== Challenge 3: Object Methods ==== 
-// Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+Intern.prototype.multiply = function(a, b) {
```

Comment:
Awesome use of a class constructor and prototypes on a day 1 project. Good stuff.